puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/tests/katello-webpack-pr-test.yaml

```diff
@@ -0,0 +1,22 @@
+- job:
+    name: katello-webpack-pr-test
+    project-type: pipeline
+    concurrent: true
+    properties:
+      - github:
+          url: https://github.com/Katello/katello
+      - tfm-pull-request-build-discarder
+    triggers:
+      - github_pr_katello:
+          context: 'katello/webpack'
+          included_regions:
+            - ^webpack\/.+
+            - ^engines\/bastion_katello\/.+
+            - ^app\/assets\/.+
+            - package.json
```

Comment:
@jlsherrill @waldenraines -- This represents folders that if they are the only things changed will trigger a build. In this case, the idea being these trigger the new katello-webpack-pr-test for ruby tests. Can you check accuracy?